src/st_stuff.c

```diff
@@ -677,6 +681,64 @@ void ST_updateFaceWidget(void)
 }
 
 int sts_traditional_keys; // killough 2/28/98: traditional status bar keys
+int hud_blink_keys; // [crispy] blinking key or skull in the status bar
+
+void ST_SetKeyBlink(player_t* player, int blue, int yellow, int red)
+{
+  int i;
+  const int keys[3] = { blue, yellow, red };
```

Comment:
This is the third time that I read through the code and my brain misparses this as some kind of `enum`. Could you add a short comment like "init with args" for me, please?

Reply:
`// Init array with args to iterate through`?

Reply:
Longer comments belong one line above please.

Reply:
I'll take that as a yes to my suggestion. Done in cb1861ff0461b37b7d85d325e56fe2bba5777cf7.